lib/generate.js

```diff
@@ -0,0 +1,41 @@
+/**!
+ * Copyright (c) 2019-2022 Digital Bazaar, Inc. All rights reserved.
+*/
+'use strict';
+
+/**
+ * Generates the Report using a set of report files.
+ */
```

Comment:
Should this be here?

Reply:
I removed it: https://github.com/digitalbazaar/mocha-w3c-interop-reporter/pull/10/commits/a59ce62571567111307ffaf5d7439ed0da5fc2fd

looks like that was a jsdoc comment for the module itself.